packages/fx-core/src/common/spec-parser/constants.ts

```diff
@@ -4,6 +4,14 @@
 
 export class ConstantString {
   static readonly CancelledMessage = "Operation cancelled";
+  static readonly SpecVersionNotSupported = "OpenAPI version lower than 3.0.0 is not supported";
+  static readonly NoServerInformation =
+    "No server information found in the OpenAPI specification file";
+  static readonly MultipleServerInformation =
+    "Multiple server information found in the OpenAPI specification file";
+  static readonly RemoteRefNotSupported = "Remote reference is not supported: %s";
+  static readonly MissingOperationId = "Missing operationId: %s";
+  static readonly NoSupportedApi = "No supported API found in the OpenAPI specification file";
```

Comment:
Should we localize these strings?

Reply:
Yeah, this is a good question and I want to discuss it in the next meeting, and if it needs localization, I can change the code in the future PR